fs/dtype_test.go

```diff
@@ -0,0 +1,18 @@
+package fs
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+)
+
+func TestRequiresRootNOP(t *testing.T) {
+
+	// This is a dummy test case that exist to call
```

Comment:
i'm currently synced up with master and using go 1.8.  if i remove fs/dtype_test.go file then i see the following failure:

    # go test github.com/containerd/containerd/fs -test.root
    flag provided but not defined: -test.root
    ....

i agree with you that testutils references test.root on all platforms.

but afaik the problem is that the only reference to the testutils package from the fs package is in dtype_linux_test.go.  so on non-linux platforms, the fs package doesn't reference the testutils package, so it doesn't define test.root.  i added fs/dtype_test.go to ensure that the fs package references the testutils package for all platforms.  that said, i'm new to go so my analysis may be incorrect or i may be overlooking a better way to fix this.